src/components/BookTree/BookTree.js

```diff
@@ -8,16 +8,25 @@ import { GET_USER_COOKBOOKS } from '../../queries/Queries'
 import { Link } from 'react-router-dom'
 
 
-const BookTree = () => {
+
+const BookTree = ({ user }) => {
   // Get user would go first to get an id could be done on a app load as well
-  const id = '1'  // Will need to pass down the user id from App 
+  // Will need to pass down the user id from App 
+  //>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+  // NEED TO DEBUG THIS. WORKS WITH '1', BUT ERRORS WITH user.id. I think it's an async issue.
+
+  const id = user.id || '0'
```

Comment:
This part is very change, we are getting a post error if we do not have the conditional

Reply:
very strange*